<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signal Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true"> QGroupBox {
    background-color: white;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="stop_button">
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: white; }</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/stop.png</normaloff>:/img/stop.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="shortcut">
            <string>Alt+S</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="play_button">
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: white; }</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/play.jpg</normaloff>:/img/play.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="shortcut">
            <string>Alt+P</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="zoom_out">
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: white; }</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/zoomout.JPG</normaloff>:/img/zoomout.JPG</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="shortcut">
            <string>Ctrl+O</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="zoom_in">
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: white; }</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/zoomin.JPG</normaloff>:/img/zoomin.JPG</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="shortcut">
            <string>Ctrl+I</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="up_button">
           <property name="text">
            <string>up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="down_button">
           <property name="text">
            <string>down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="speed_signal">
           <item>
            <property name="text">
             <string>speed_signal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>50</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>80</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="show_signals">
           <property name="text">
            <string>show</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hide_signals">
           <property name="text">
            <string>hide</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="spectrogram">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="PlotWidget" name="graphicsView_1">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::NoDrag</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSlider" name="max_contrast">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>vmin</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="max_contrast_2">
      <property name="text">
       <string>vmax</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSlider" name="min_contrast">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSlider" name="right_scroll">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSlider" name="left_scroll">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_Signals"/>
    <addaction name="actionPrint_to_PDF"/>
    <addaction name="actionClearAll"/>
   </widget>
   <widget class="QMenu" name="menuChannels">
    <property name="title">
     <string>Signals</string>
    </property>
    <addaction name="actionSignal_1"/>
    <addaction name="actionSignal_2"/>
    <addaction name="actionSignal_3"/>
    <addaction name="actionspectrogram"/>
    <addaction name="actionchannel_2"/>
   </widget>
   <widget class="QMenu" name="menuColor">
    <property name="title">
     <string>Color</string>
    </property>
    <addaction name="actionPalette"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuChannels"/>
   <addaction name="menuColor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPrint_to_PDF">
   <property name="text">
    <string>Print to PDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClearAll">
   <property name="text">
    <string>Clear all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionchannel_1">
   <property name="text">
    <string>Channel 1</string>
   </property>
  </action>
  <action name="actionChannel_2">
   <property name="text">
    <string>Channel 2</string>
   </property>
  </action>
  <action name="actionChannel_3">
   <property name="text">
    <string>Channel 3</string>
   </property>
  </action>
  <action name="actionAdd_Signals">
   <property name="text">
    <string>Add Signals</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionChannel_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Channel 1</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionChannel_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Channel 2</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionspectrogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>spectrogram</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionPalette">
   <property name="text">
    <string>Palette</string>
   </property>
  </action>
  <action name="actionblue">
   <property name="text">
    <string>blue</string>
   </property>
  </action>
  <action name="actionyellow">
   <property name="text">
    <string>yellow</string>
   </property>
  </action>
  <action name="actiongreen">
   <property name="text">
    <string>green</string>
   </property>
  </action>
  <action name="actionpink">
   <property name="text">
    <string>pink</string>
   </property>
  </action>
  <action name="actionSignal_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Signal 1</string>
   </property>
  </action>
  <action name="actionSignal_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Signal 2</string>
   </property>
  </action>
  <action name="actionSignal_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Signal 3</string>
   </property>
  </action>
  <action name="actionchannel_2">
   <property name="text">
    <string>channel_1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
